intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/formatter/settings/YangCodeStyleMainPanel.java

```diff
@@ -0,0 +1,20 @@
+package tech.pantheon.yanginator.plugin.formatter.settings;
+
+import com.intellij.application.options.TabbedLanguageCodeStylePanel;
+import com.intellij.psi.codeStyle.CodeStyleSettings;
+import tech.pantheon.yanginator.plugin.YangLanguage;
+
+public class YangCodeStyleMainPanel extends TabbedLanguageCodeStylePanel {
+    public YangCodeStyleMainPanel(final CodeStyleSettings currentSettings,
```

Comment:
Could you please separate the class header from the constructor (or whatever else is there) with a blank line ? It's easier on eyes that way.
The same applies to other classes in this patch.